src/cap-description.adoc

```diff
@@ -521,8 +521,9 @@ malformed    =  !EF && (malformedMSB || malformedLSB)
 NOTE: The check is for malformed _bounds_, so it does not include reserved
 bits!
 
-Capabilities with malformed bounds are always invalid anywhere in the system
-i.e. their tags are always 0.
+It is impossible for a CHERI core to generate a tagged capability with malformed
+bounds, or with any reserved bits set. If such a capability exists then it must
+have been caused by a logic or memory fault.
```

Comment:
How about adding a note here that if the bounds of such a capability are requested they must decode as 0/0 and address manipulation instructions (CADD(I)) detag the result when such a capability is received as input.

Reply:
ok - done - please have another look